src/containers/HubiiApiHoc/saga.js

```diff
@@ -39,16 +39,16 @@ import {
 } from './actions';
 
 
-export function* loadWalletBalances({ address, noPoll }, network) {
+export function* loadWalletBalances({ address, noPoll }, _network) {
+  let network = _network;
```

Comment:
would it make sense to replace this line with `let network = _network || yield  select(makeSelectCurrentNetwork())` and remove the line `if (noPoll) network = yield select(makeSelectCurrentNetwork());`?

Reply:
Good idea, will make the change